shell/platform/linux/fl_window_state_monitor_test.cc

```diff
@@ -0,0 +1,255 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/linux/fl_window_state_monitor.h"
+#include "flutter/shell/platform/linux/public/flutter_linux/fl_string_codec.h"
+#include "flutter/shell/platform/linux/testing/fl_test.h"
+#include "flutter/shell/platform/linux/testing/mock_binary_messenger.h"
+#include "flutter/shell/platform/linux/testing/mock_window.h"
+
+#include "gtest/gtest.h"
+
+class LifecycleStringMatcher {
```

Comment:
Add a doc comment.

Reply:
This is cargo culted from fl_platform_handler_test.cc which doesn't have a comment :)

Reply:
These feel like they should be common code somewhere.